src/components/date/date.js

```diff
@@ -120,12 +120,12 @@ const Date = Input(InputIcon(InputLabel(InputValidation(class Date extends React
     allowEmptyValue: PropTypes.bool,
 
     /**
-     * Display the date picker component on top of the input
+     * Choose where displayed the date picker
      *
-     * @property showPickerOnTop
+     * @property positionDatePicker
      * @type {Boolean}
      */
-    showPickerOnTop: PropTypes.bool
+    positionDatePicker: PropTypes.string
```

Comment:
You could set this to `positionDatePicker: PropTypes.oneOf(OptionsHelper.positionDatePicker)` to limit the accepted prop values to the constants you created